addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,17 +2,14 @@
 <templates xml:space="preserve">
 
     <t t-name="web.NotificationWowl" owl="1">
-        <div t-attf-class="o_notification {{className}}"
-        role="alert" aria-live="assertive" aria-atomic="true">
-            <div class="o_notification_body">
-                <button type="button" class="close o_notification_close"
-                aria-label="Close" t-on-click="props.close">
-                    <span class="d-inline" aria-hidden="true">×</span>
-                </button>
-                <div t-if="props.title" class="font-weight-bold o_notification_title mb-1" t-esc="props.title"/>
-                <div t-if="props.message" class="mr-auto o_notification_content">
-                    <t t-out="props.message"/>
-                </div>
+        <div t-attf-class="o_notification border border-{{props.type}} bg-white mb-2 position-relative"
+         role="alert" aria-live="assertive" aria-atomic="true">
+            <strong t-if="props.title" t-attf-class="o_notification_title d-block text-{{props.type}} py-2 pl-3 pr-4" t-out="props.title"/>
+            <button type="button" class="o_notification_close close" aria-label="Close" t-on-click="props.close">
```

Comment:
black borders are shown when this element is :active

Reply:
Actually, those borders are both present on `master` and even in the Bootstrap's "Close Icon" (cf. https://getbootstrap.com/docs/4.3/utilities/close-icon/)

Reply:
Yes, it's conceived like that intentionally in order to to hilight the button when on `:focus`.

The fact is that we don't have hotkeys targeting it (or notifications in general neither).
Also, notifications don't get focus when rendered, so you can't [TAB] to select the button. 

I'd suggest to apply a `btn` class and remove the outline effect.
With  the current implementation the outline is just annoying and don't add any value IMO